ipynb/examples/utils/executor_example.ipynb

```diff
@@ -1,14 +1,39 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Executor API - Executor\n",
+    "\n",
+    " A tests executor is a module which supports the execution of a configured set of experiments.<br><br>\n",
+    " Each experiment is composed by:\n",
+    "  - a target configuration\n",
+    "  - a worload to execute\n",
+    "\n",
+    "The executor module can be configured to run a set of workloads (wloads) in each different target configuration of a specified set (confs). These wloads and confs can be specified by the \"tests_config\" input dictionary which is described below at **Tests Configuration**.<br><br>\n",
```

Comment:
`tests_config` is called `experiments_config` now (to disambiguate it from the `test_config` that configures the TestEnv :confused: